ComicFormViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ComicFormViewModel.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/13/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+/// Specifies whether the ComicFormViewController will "add" comic books or "edit" comic books in a users collection.
+enum  ComicFormMode {
+    case add
+    case edit
+}
+
+struct ComicFormViewModel {
+    var coverPhoto: UIImage?
+    var comicFormMode: ComicFormMode = .add
+    var seriesTitle: String?
+    var volume: String?
+    var storyTitle: String?
+    var issue: String?
+    var publisher: String?
+    var release: String?
+    var genre: Comic.Genre?
+    var condition: Comic.Condition?
+
+    // ComicFormMode is set to .add by default if default init is used.
+    init() {}
+
+    /// Initializes ComicFormViewModel with a ComicFormMode of edit.
+    /// The parameters will be used by the ComicFormViewController to fill in its textfields.
+    init(coverPhoto: UIImage?, seriesTitle: String, volume: String?, storyTitle: String, issue: String?, publisher: String?, release: String?, genre: Comic.Genre?, condition: Comic.Condition?) {
```

Comment:
All parameters but the first should be on their own lines indented to the level of the first e.g.
```
init(coverPhoto: UIImage?, 
     seriesTitle: String, 
     volume: String?,
     ...
     condition: Comic.Condition?) {
```